src/app/pages/learning-research/components/publications.js

```diff
@@ -0,0 +1,79 @@
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faArrowUpRightFromSquare} from '@fortawesome/free-solid-svg-icons/faArrowUpRightFromSquare';
+
+export const Publications = ({data: {publicationsHeader, publications}}) => {
+    const [viewAll, setViewAll] = useState(false);
+    const initialCount = isMobileDisplay() ? 3 : 5;
+    const publicationList = viewAll ? publications : publications.slice(0, initialCount);
+
+    const publicationsRef = useRef(null);
+    const scrollToPublications = () => {
+        publicationsRef.current?.scrollIntoView({ behavior: 'smooth' });
+    };
+
+    return (
+        <Section>
+            <Box direction='column' ref={publicationsRef}>
+                <h2>{publicationsHeader}</h2>
+                {publicationList.map((publication, index) =>
+                    <PublicationItem key={index} publication={publication} />
+                )}
+                <span
+                    className='py-4'
+                    onClick={() => {
+                        if (viewAll) {
+                            scrollToPublications();
+                        }
+                        setViewAll(!viewAll);
+                    }}
+                    css={{
+                        cursor: 'pointer',
+                        color: colors.linkText,
+                        alignSelf: 'center'
+                    }}>
+                    <FontAwesomeIcon icon={viewAll ? faChevronUp : faChevronDown} />
+                    &nbsp;
+                    {viewAll ? 'View Less' : 'View All Publications'}
+                </span>
+            </Box>
+        </Section>
+    );
+};
+
+export const PublicationItem = ({ publication }) => (
+    <Box direction='column' className='py-2' css={{ lineHeight: 1.8 }}>
+        <div>
+            <a className='large-link' href={publication.pdf} target='_blank' rel="noreferrer">
+                {publication.title}
+            </a>
+        </div>
+        <div>
+            <span>{publication.date} &middot;&nbsp;</span>
+            <span css={{ color: colors.grayText }}>
+                {publication.date} {publication.authors} {publication.excerpt}
+            </span>
+        </div>
+        <Box gap='xlarge'>
+            <a className='text-decoration-none' href={publication.pdf} target='_blank' rel="noreferrer">
+                <Box align='center'>
+                    Pdf&nbsp;
+                    <FontAwesomeIcon size='sm' icon={faArrowUpRightFromSquare} />
+                </Box>
+            </a>
+            {publication.github &&
+                <a className='text-decoration-none' href={publication.github} target='_blank' rel="noreferrer">
+                    <Box align='center'>
+                        Github&nbsp;
+                        <FontAwesomeIcon size='sm' icon={faArrowUpRightFromSquare} />
+                    </Box>
+                </a>}
+        </Box>
```

Comment:
using a run-on non breaking space makes the words a bit hard to read here.  Maybe it'd be better to wrap the text in a span and set `gap` on the wrapping Box.

```jsx
<Box align='center' gap>
  <span>Github</span>
  <FontAwseomeIcon ... />
</Box>
```